src/Components/Blazor/Build/src/ReferenceBlazorBuildFromSource.props

```diff
@@ -0,0 +1,25 @@
+<Project>
+  <!--
+  Importing this file is equivalent to having:
+    <PackageDependency Include="Microsoft.AspNetCore.Blazor.Build" />
+  ... except it's much more convenient when working in this repo, because it consumes the
+  Blazor.Build targets/exe directly without needing this project to be packed into a .nupkg.
+
+  This is only intended for use by other projects in this repo.
+  -->
+
+  <PropertyGroup>
```

Comment:
We want to avoid referencing any projects \ sources from the test app. Doing so lets us avoid arcade and build the test app from temp directories outside AspNetCore src tree

Reply:
I'm not sure I get this comment.

Reply:
`ReferenceFromSource.props` adds project references - https://github.com/aspnet/AspNetCore/blob/master/src/Components/Blazor/Build/src/ReferenceFromSource.props#L39-L55 - which we want to avoid. All this wires up is build targets and tasks that can be safely referenced by test apps building outside Arcade \ AspNetCore.